src/fprime/fpp/visualize.py

```diff
@@ -146,9 +146,9 @@ def add_fpp_viz_parsers(
         default=7000,
     )
     viz_parser.add_argument(
-        "--cache-dir",
+        "--working-dir",
         help="Set the directory to store layout files in [default: %(default)s]",
         required=False,
-        default=".visualize-cache",
+        default="/tmp/fprime-visualize",
```

Comment:
What happens when this directory does not give write permissions?  Do we tell the user that they should override with the `--working-dir` flag.

Reply:
Good point, I added a more verbose error with resolution approach.